testdata/test

```diff
@@ -2,8 +2,8 @@ show-widgets
 showWidgets
 show_widgets
 show-widgets
-old-pricing-banner
 oldPricingBanner
 show_widgets
 oldPricingBanner
 mobile-app-promo-ios
+mobile-app-promo-ios
```

Comment:
looks like this is listed twice now?

Reply:
yea just test data to show that everything is working properly - this file changes with most PRs